source/session.js

```diff
@@ -825,9 +829,13 @@ export class Session {
     let headers;
 
     const updateOnProgressCallback = (oEvent) => {
+      let progress = 0;
```

Comment:
jsdom won't set request headers before `request`, cause `onProgress` never be called in test enviroment.